Documentation/ceph-quickstart.md

```diff
@@ -16,7 +16,7 @@ from other pods running in your cluster.
 
 Kubernetes **v1.11** or higher is supported by Rook.
 
-**Important** If you are using K8s 1.15 or older, you will need to create a different version of the Rook CRDs. Create the `crd.yaml` found in the [pre-k8s-1.16](https://github.com/rook/rook/blob/{{ branchName }}/cluster/examples/kubernetes/ceph/pre-k8s-1.16) subfolder of the example manifests.
+**Important** If you are using K8s 1.15 or older, you will need to create a different version of the Rook CRDs. Create the `crds.yaml` found in the [pre-k8s-1.16](https://github.com/rook/rook/blob/{{ branchName }}/cluster/examples/kubernetes/ceph/pre-k8s-1.16) subfolder of the example manifests.
```

Comment:
Have you rebased since #6556 was merged? The renaming of crd.yaml to crds.yaml was done there.

Reply:
Looks like I had forgotten. Done now.